### ISSUE TITLE
Getting different result while training an mlp with LSTM layer on Keras

### ISSUE THREAD

**Issue by sharod** [keras-team/keras#7681]
I am using keras with tensorflow backend. I am creating a simple 1 layer LSTM model. I need my code to give same val_loss every time I train on the same data. I am running my system on CPU only. I tried:

from numpy.random import seed
seed(1337)
from tensorflow import set_random_seed
set_random_seed(1337)

on the top of my code.
I also initialized all the kernels and recurrent as ones:

model.add(LSTM(150,input_shape=(None,124), W_regularizer=l2(0.001),kernel_initializer='ones', recurrent_initializer='ones', bias_initializer='ones'))
model.add(Dense(2,kernel_initializer='ones', bias_initializer='ones'))
model.add(Activation("softmax"))

I also set shuffle=False in model.fit(). I am using rms for optimizing.

I also set PYTHONHASHSEED to 0. But still I am getting different train loss as well as val loss on each epoch when rum multiple times.

I  am running keras on a server with 56 cpu cores and CentOS.

Pls help soon as I have tried everything everyone has suggested in other threads !!!!!
